app/views/users/show.html.erb

```diff
@@ -1 +1,36 @@
-<h1>Show details of a particular user</h1>
\ No newline at end of file
+<header>
+    <%= stylesheet_link_tag 'users_show' %>
+</header>
+
+<div class="all-users">
+    <div class="user-show-container">
+        <div class="photo-container">
+            <img src="<%= @user.photo %>" alt="user photo" class="user-photo">
+        </div>
+        <div class="user-section">
+            <div class="username"><%= @user.name %></div>
+            <div class="post-count">Number of posts: <%= @user.postcounter %></div>
+        </div> <br>
+    </div>
+    <div class="bio-section">
+        <div class="bio">
+            Bio
+        </div>
+        <div class="bio-text">
+            <%= @user.bio %>
+        </div>
+    </div>
+    <% @ordered_post = @user.posts.order(created_at: :asc).limit(3) %>
+    <% @ordered_post.each do |post| %>
+    <div class="bio-section">
```

Comment:
- [ ] Once again instead of manually writing out the logic for fetching the 3 most recent posts I'd suggest you use the `three_most_recent_posts` method from the `User` model as specified in the project requirements.